components/AudioControls.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <v-container fluid>
+    <v-row>
+      <v-col cols="9" md="3" class="d-flex flex-row pa-0">
+        <v-avatar ref="albumCover" tile size="72" color="primary">
+          <v-img :src="getImageUrl(getCurrentItem.AlbumId)">
+            <template #placeholder>
+              <v-icon dark>mdi-album</v-icon>
+            </template>
+          </v-img>
+        </v-avatar>
+        <div class="d-flex flex-column justify-center ml-4">
+          <span class="font-weight-medium mt-md-n2">
+            <nuxt-link
+              tag="span"
+              class="text-truncate link"
+              :to="`/item/${getCurrentItem.AlbumId}`"
+              >{{ getCurrentItem.Name }}</nuxt-link
+            >
+            <v-btn class="d-none d-md-inline-flex" icon>
+              <v-icon size="18">{{
+                getCurrentItem.UserData.IsFavorite
+                  ? 'mdi-heart'
+                  : 'mdi-heart-outline'
+              }}</v-icon>
+            </v-btn>
+          </span>
+          <nuxt-link
+            tag="span"
+            class="text--secondary text-caption text-truncate mt-md-n2 link"
+            :to="`/artist/${getCurrentItem.AlbumArtists[0].Id}`"
```

Comment:
Isn't this making the assumption that there is an album artist.

In the case where there isn't, won't this produce an error

Reply:
I always have an album artist in my library, even for albums from solo artists.